pylint/extensions/magic_value.py

```diff
@@ -37,11 +38,30 @@ class MagicValueChecker(BaseChecker):
                 "default": (0, -1, 1, "", "__main__"),
                 "type": "csv",
                 "metavar": "<argument names>",
-                "help": " List of valid magic values that `magic-value-compare` will not detect.",
+                "help": " List of valid magic values that `magic-value-compare` will not detect,"
+                "Supports integers, floats, negative numbers, for empty string enter ``''``,"
+                " for backslash values just use one backslash e.g \\n.",
             },
         ),
     )
 
+    def __init__(self, linter: PyLinter) -> None:
+        """Initialize checker instance."""
+        super().__init__(linter=linter)
+        self.valid_magic_vals: tuple[(float | str), ...] = ()
```

Comment:
I have never seen this syntax.. Shouldn't it be:
```suggestion
        self.valid_magic_vals: tuple[float | str, ...] = ()
```